project-koku/masu
# Summarizing OCP storage line item data

**Overview**
Summarizing OCP storage line item data.  The remaining summary tables will be done in a follow on PR.

**Testing**
1.  Add OCP provider to koku
2.  Upload OCP payload with storage and cpu/mem usage reports.
3. Ingest data with masu download endpoint.
4.  Verify that the correct summary db tables are updated.

**Test Results**
[masu_ocp_storage_lineitems_ut.txt](https://github.com/project-koku/masu/files/2853945/masu_ocp_storage_lineitems_ut.txt)
